docs/sources/metainfo/component.xml

```diff
@@ -646,6 +650,34 @@
 				</listitem>
 			</varlistentry>
 
+			<varlistentry>
+				<term>&lt;input/&gt;</term>
+				<listitem>
+				<para>
+					Specifies that a input capability type is supported by the hardware present in the device.
+					Options here are: keyboard, pointer, touch, tablet-tool, tablet-pad, gesture, switch
+					<!-- As listed here https://wayland.freedesktop.org/libinput/doc/latest/group__device.html#ga3110cdddce94a1df0b8a3306909c8f15 -->
```

Comment:
Nice, we should have a list of allowed values based on the libinput input types here.

Reply:
That's what we have after "options here are:", no?